README.rst

```diff
@@ -31,6 +31,7 @@ UCI configuration options must go in ``/etc/config/monitoring``.
 
 - ``monitored_interfaces``: interfaces that needs to be monitored, defaults to ``*`` for all interfaces.
 - ``interval``: time after which device data should be sent to server, defaults to ``300``
+- ``required_memory``: free memory required to save data temporarily, defaults to ``0.05``
```

Comment:
is it free or available memory?

Reply:
Free memory

Reply:
there's a problem with this: on modern systems it will always look like the free memory is almost nonexistent.
Linux allocates almost the entire memory to the operating system, ready to be used if needed.
That's why on modern systems you will see that there's also another metric called "available memory".
We should try to use available memory and use free memory only if missing.
Otherwise, we risk this work to be uneffective, because the daemon will think the memory is full and will not store the data.

Reply:
My bad it is available memory. I mentioned free here as that memory used can be freed when needed :thinking: